projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts

```diff
@@ -25,44 +35,186 @@ context('Quote', () => {
       },
     };
     cy.cxConfig(globalMessageSettings);
+    quote.log('visit default url', 'beforeEach');
     cy.visit('/');
-    quote.login(EMAIL, PASSWORD, USER);
+    quote.log(`login as ${BUYER_USER}`, 'beforeEach');
+    quote.login(BUYER_EMAIL, BUYER_PASSWORD, BUYER_USER);
+    quote.log('registerGetQuoteRoute', 'beforeEach');
     quote.registerGetQuoteRoute(POWERTOOLS);
   });
 
   describe('Request quote process', () => {
     it('should display a message and disable submit button if threshold is not met', () => {
-      quote.requestQuote(POWERTOOLS, testProductHammerDrillingId, '1');
-      quote.checkQuoteInDraftState(false, testProductHammerDrillingId);
+      quote.log('request a quote', getTestTitle());
+      quote.requestQuote(POWERTOOLS, TESTPRODUCTHAMMERDRILLINGID, '1');
```

Comment:
same here move the log into the helper method and merge it with existing log in the helper method, so there is only one log

Reply:
done